bootstrap/terraform/values/argocd.yaml

```diff
@@ -158,6 +187,39 @@ configs:
             message = "Provisioning ..."
           }
 
+          local function contains (table, val)
+            for i, v in ipairs(table) do
+              if v == val then
+                return true
+              end
+            end
+            return false
+          end
+
+          local has_no_status = {
+            "Composition",
+            "CompositionRevision",
+            "DeploymentRuntimeConfig",
+            "ControllerConfig",
```

Comment:
Same question `deploymentruntimeconfigs.pkg.crossplane.io` and `controllerconfigs.pkg.crossplane.io` not `*.aws.crossplane.io`?